rpc/src/v1/impls/signing.rs

```diff
@@ -114,6 +115,12 @@ impl<D: Dispatcher + 'static> SigningQueueClient<D> {
 
 	fn dispatch(&self, payload: RpcConfirmationPayload, origin: Origin) -> BoxFuture<DispatchResult> {
 		let default_account = self.accounts.default_account();
+		let from = &payload.sender().unwrap_or(&default_account);
+		// bail early if the account isn't unlocked
+		if !self.accounts.is_unlocked(from) && !self.signer.is_enabled() {
```

Comment:
@seunlanlege 

I don't think this will work because `self.accounts.is_unlocked(from)` and `self.signer.is_enabled()` are mutable exclusive i.e, if no account is unlocked then it will regard it as an external signer.

Thus, the if above will never succeed AFAIK, can you convince me that I'm wrong? By either explain how you tested it or add an additional test for this?

Reply:
I've added a test that affirms the new behaviour.

Reply:
Thank you for the added test, but it doesn't really test what I was referring to sorry for being a PITA. 

However,  [support_token_api](https://github.com/paritytech/parity-ethereum/blob/master/parity/configuration.rs#L899-#L901) is only enabled when the `account is unlocked` which is passed to the [signer service](https://github.com/paritytech/parity-ethereum/blob/master/parity/signer.rs#L37-#L41)

So, how will/can it work?

Reply:
Great catch, for some reason it's disabled in the test enviroment. I've decided to add a cli argument to enable the signer queue.